bridge-token/src/lib.rs

```diff
@@ -60,6 +60,16 @@ impl BridgeToken {
         }
     }
 
+    pub fn assert_caller_self_or_factory(&mut self) -> bool {
+        if env::predecessor_account_id() == self.controller ||
+        env::predecessor_account_id() == env::current_account_id()
+        {
```

Comment:
```suggestion
        if env::predecessor_account_id() == self.controller 
            || env::predecessor_account_id() == env::current_account_id() {
```

The indentation and stylistic are wrong.